packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -312,7 +312,10 @@ const INDIVIDUAL_ACTIONS = [
     level: 'primary',
   },
   {
-    handler: (remote, { editRemote }) => editRemote(remote),
+    handler: (remote, { reset, editRemote }) => {
+      reset()
```

Comment:
Only reset if the edit succeeds.

Reply:
This is not the `editRemote` that will make the call to the server. It's the one that will change the component's state to populate the form with the remote's information so that the user can edit it. Maybe the name is confusing, then.